CONTRIBUTING.md

```diff
@@ -1,31 +1,37 @@
 # Contributing to Riot Examples
 
-If you’d like to contribute a feature or bug fix, you can fork Riot, commit your changes, & send a pull request. Please make sure to search the issue tracker first.
+If you'd like to contribute an example or bug fix, feel free to send us a pull
+request. Please make sure to search the issue tracker first for duplicates.
```

Comment:
It seems better without `new line`?
GitHub Markdown replaces one with `<br>`, two with `<p>`.
https://help.github.com/articles/writing-on-github/#newlines